<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDRechercherUnGroupe</class>
 <widget class="QDialog" name="BDRechercherUnGroupe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Groupes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_en_cours">
        <property name="text">
         <string>En cours</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rb_tout">
          <property name="text">
           <string>Recherche spécifique  :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelNomGroupe">
            <property name="text">
             <string>Nom du groupe :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditNomGroupe"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_rechercherGroupe">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTreeWidget" name="treeWidgetGroupes">
        <attribute name="headerDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>IdGroupe</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Groupe</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Responsable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type de client</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nb pers prévues</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nb pers enregistrées</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type de chambre maj</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date d'arrivée</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Durée</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Avance</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant total</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant payé</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant restant</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_imprimerHotel">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Imprimer facture hôtel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_imprimerBar">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Imprimer facture Bar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_imprimerResto">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Imprimer facture Resto</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btn_contacts">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Voir contacts</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout_3">
     <property name="formAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_choisir">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Choisir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_annuler">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
